packages/downloads/test/utils.test.ts

```diff
@@ -0,0 +1,23 @@
+import { urlBuilder } from "../src/utils";
+
+describe("utils", () => {
```

Comment:
Although trivial, and coverage is provided by other tests, `composeDownloadId` is a function exported from utils, so it would be good to have a test here

Reply:
Added in c735ae5.